### ISSUE TITLE
tivimate error code 524

### ISSUE THREAD

**Issue by widdlepear** [dtankdempse/daddylive-m3u#20]
Do you have any idea what this means?  Tivimate was working fantastically until this code now pops up on all my tv's.  Thanks so much for your help and all your hard work even if you can't help me.

**Comment by dtankdempse** [comment]
Not sure, I use TiviMate also and I am not getting a 524. What version are you running? Are you using the m3u-playlist-proxy at all?

**Comment by widdlepear** [comment]
Im using Tivimate Version 5.1.0 (Premium).  No I am not using the playlist proxy.  Should I be?  If so is could you point to reading material how to proceed.  I am sorry to be a noob and ask silly questions.  But I really appreciate your time either way.  Your work is amazing and I am extremely thankful for you even responding.

**Comment by dtankdempse** [comment]
No, you don’t need to use the proxy with Tivimate. I just wanted to confirm you weren’t using it. I reverted to version 5.0.4, which fixed many of the issues I encountered in version 5.1.0. If you have an older device to test the rollback, check out this thread [here](https://www.reddit.com/r/TiviMate/comments/1ffdv68/tivimate_roll_back/). Let me know how it goes if you decide to try reverting back.

**Comment by widdlepear** [comment]
Great help...so appreciative. Reverted to 5.0.4 and its back to normal.  It must have auto updated.  I turned off the auto update.  Fingers crossed.

**Comment by widdlepear** [comment]
Hey thanks again @dtankdempse...Your idea worked like a charm, can i make a small contribution to your cause?

**Comment by dtankdempse** [comment]
Thank you for the offer! I haven't set anything up yet, as I wasn't too concerned about donations. I might consider it later, but i genuinely appreciate your support. :)
